### PR TITLE
fix: dependency

### PR DESCRIPTION
**Pull request by dte**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,5 +42,8 @@
     "webpack-cli": "^3.3.0",
     "webpack-config-utils": "^2.3.1",
     "webpack-dev-server": "^3.2.1"
+  },
+  "dependencies": {
+    "lodash": "^4.17.15"
```
Comment (mlunoe) [inline package.json]: I don't see where this dependency is being used?

Comment (dte) [reply package.json]: weird. I was seeing an error in Github for a vulnerable dependency. https://github.com/Paperspace/client-sdk/network/alerts

Comment (dte) [reply package.json]: maybe  a child dependency?

Comment (dte) [reply package.json]: babel maybe: https://github.com/Paperspace/client-sdk/blob/master/yarn.lock#L25

Comment (mlunoe) [reply package.json]: Ah, ok. Then we should upgrade babel core (along with the rest of the babel suite - they follow dependency numbering)

**Pull request closed without merging**